packages/react-cli/src/commands/generate.ts

```diff
@@ -16,17 +16,15 @@ export class GenerateCommand extends Command {
     '$ react generate route route-name',
     '$ react generate route path/to/route-name',
 
-    '$ react generate component component-name --route=dashboard/posts'
+    '$ react generate component component-name --route=dashboard/posts',
   ];
 
-  static args = [
-    { name: 'generator', required: true },
-    { name: 'name', required: true },
-  ];
+  static args = [{ name: 'generator', required: true }, { name: 'name', required: true }];
 
   static flags = {
     route: flags.string({
-      description: 'directory of a route to place the component in. Will live in `route-path/-components/`',
+      description:
+        'directory of a route to place the component in. Will live in `route-path/-components/`',
```

Comment:
![image](https://user-images.githubusercontent.com/7727018/56309220-166e9a00-6117-11e9-892b-9aa928347d15.png)

the leading hyphen confused me a bit, is this the intended folder name?

Reply:
yup, since file-systems sort alphabetically, this puts your components at the top.

so, in any given sub-route, you'll have this:
```
-components/
sub-routeA/
sub-routeB/
sub-routeC/
index.tsx
```

Reply:
Oh that makes sense!